src/OpenTelemetry.ResourceDetectors.Azure/CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
 # Changelog
 
-## Unreleased
+## 1.0.0-beta.3
 
+Released 2023-Sep-19
+
+* Added a timeout for `HttpClient` for outgoing call made to metadata service to
```

Comment:
please add the timeout value here itself, so users does not have to open the PR and then the code in it to understand the actual value.

Reply:
done